src/Oriflame.RedisMessaging.ReliableDelivery/Subscribe/MessageProcessor.cs

```diff
@@ -9,6 +9,11 @@ namespace Oriflame.RedisMessaging.ReliableDelivery.Subscribe
     /// <inheritdoc cref="IMessageDeliveryChecker" />
     internal class MessageProcessor : IMessageProcessor, IMessageDeliveryChecker
     {
+        /// <summary>
+        /// Threshold for difference PubSub received message time and handled message time
+        /// </summary>
+        private static readonly TimeSpan PubSubReceivedHandledThreshold = TimeSpan.FromSeconds(2);

```

Comment:
I think it culd be helpful to make the property `PubSubReceivedHandledThreshold` configurable in case the need arises. What do you think?

Reply:
fixed